payloads/data-reporting/v3.4/reports-api/responses/chats/totalChats.json

```diff
@@ -1,30 +1,31 @@
 {
-  "total": 329,
+  "total": 369,
```

Comment:
I think we could update payloads with the `name` field in this PR.

Reply:
@ale-kacperczyk done